pages/lessons_incubation.md

```diff
@@ -0,0 +1,235 @@
+    are covered, at which the lead developer(s) will usually teach the
+    entire material for early adopters to observe. Based on Data
+    Carpentry’s experience, such a hackathon costs roughly US$20,000.
+
+## 3. Roadmap
+
+Our process for new lessons is:
+
+1.  The proposer contacts the Executive Director (ED) of either or
+    both Data Carpentry and Software Carpentry by email to determine
+    whether someone is already working on a lesson in this area, and
+    if not, whether the criteria in Section 1 are satisfied.  The EDs
+    may introduce a proposer to others who have expressed an interest
+    in the same materials so that they can coordinate efforts.
+
+2.  If one or both EDs express interest, the proposer works with the
+    EDs and other members of the Software and/or Data Carpentry
+    community to develop and submit a lesson development proposal.
+    (See below for a template.)
+
+3.  Data Carpentry and/or Software Carpentry review the proposal. This
+    may include soliciting opinions from community members and domain
+    experts, or both. The review process is open: all comments will be
+    visible to all community members. Community members who wish to
+    join the proposal or support it in other ways may do so at this
+    point.
+
+4.  If Data and/or Software Carpentry accept the proposal, they
+    publish it on their website(s).
+
+5.  The proposers then begin work in an openly-accessible version
+    control repository (e.g., GitHub), reporting progress
+    monthly. They may adjust goals throughout the development process
+    based on feedback from community members; substantial changes in
+    goals must be approved by a representative of the appropriate
+    Carpentry.
+
+6.  Once the lesson leads have a minimum viable lesson, they deliver
+    it, and then provide feedback (both theirs and their learners’) to
+    Data and/or Software Carpentry. This feedback must include items
+    agreed to in the assessment plan incorporated into the Lesson
+    Development Proposal.
+
+7.  The lesson is added to AMY (the web-based tool we use to manage
+    the instructor database) so that instructors can indicate their
+    ability and willingness to teach it.
+
+8.  After the lesson leads have revised the lesson based on early
+    feedback, the Carpentries organize a hackathon to refine the
+    material and introduce it to early adopters. This hackathon will
+    usually include an on-site delivery of the material.
+
+9.  Early adopters (and the lesson leads) deliver the lesson at
+    multiple sites, coalescing feedback and refining the lesson.
+
+10. The lesson leads and Carpentry community members then publish the
+    lesson on the Carpentry websites.
+
+11. The lesson leads provide two hour-long online training sessions
+    in which they discuss the motivation for the lesson, their
+    experience developing it, and what they have learned from early
+    deliveries.
```

Comment:
Who are these training sessions delivered to?  Where are they advertized?  Is it “advertized on discuss@ (or the analagous DC channel) for attendance by the general public but pitched for existing SWC(/DC) instructors”?  Or is it pitched at existing maintainers?  Or…?